pypeit/tests/test_traceslits.py

```diff
@@ -8,135 +8,102 @@
 import glob
```

Comment:
maybe rename this file test_edgetrace.py ?

Reply:
done